data/json/itemgroups/Weapons_Mods_Ammo/ammo.json

```diff
@@ -870,8 +870,8 @@
     "entries": [
       { "item": "light_battery_cell", "prob": 25, "charges": [ 0, 100 ] },
       { "item": "light_minus_battery_cell", "prob": 12, "charges": [ 0, 50 ] },
-      { "item": "light_plus_battery_cell", "prob": 9, "charges": [ 0, 150 ] },
-      { "item": "light_disposable_cell", "prob": 50, "charges": [ 0, 300 ] }
+      { "item": "light_battery_cell", "prob": 9, "charges": [ 0, 150 ] },
+      { "item": "light_battery_cell", "prob": 50, "charges": [ 0, 300 ] }
```

Comment:
3 instances of `light_battery_cell` in this group and no `light_cell_rechargeable`

Reply:
itemgroups should probably be adressed in another PR